indigo_app/urls.py

```diff
@@ -92,6 +92,9 @@
     re_path(r'^works(?P<frbr_uri>/\S+?)/edit/$', works.EditWorkView.as_view(), name='work_edit'),
     re_path(r'^works(?P<frbr_uri>/\S+?)/edit/repeal$', works.EditWorkRepealView.as_view(), name='work_edit_repeal'),
     re_path(r'^works(?P<frbr_uri>/\S+?)/edit/parent$', works.EditWorkParentView.as_view(), name='work_edit_parent'),
+    re_path(r'^works(?P<frbr_uri>/\S+?)/edit/find-publication$', works.FindPublicationDocumentView.as_view(), name='find_publication_document'),
+    re_path(r'^works(?P<frbr_uri>/\S+?)/edit/attach-publication$', works.WorkFormPublicationDocumentView.as_view(), name='attach_publication_document'),
+    re_path(r'^works(?P<frbr_uri>/\S+?)/edit/localities-options$', works.WorkFormLocalityOptionsView.as_view(), name='locality_options'),
```

Comment:
The pattern used elsewhere is that the URL is the name of the field, such as `parent`, `repeal` -- in this case it would be `locality`. The options is neither here nor there.

Reply:
@longhotsummer I've moved these views to be under the place view. This allows them to work while in whether adding a new work or editing an existing one. In places where we need to reference the work I've added a get parameter with the frbr uri,